csun-metalab/faculty-v3
# Updates to Citations Tab

In this PR:
- Added a lot more data to each citation-card (now displays citation abstract, collaborators, date, etc)
- Turned each citation-card into an accordion (in order to house all of the new data)
- Sort citation-cards by year (newest first)
- Modify some CSS/Sass to make styling of "cards" on citation tab & profile tab modular/unified

To test:
- Go to someone's "citations" tab (try `deborah.chen` and `steven.fitzgerald` or anybody else)
- Check that there are no console errors
- Check that it looks visually pleasing (enough) in all breakpoints
- Check that everything makes sense on each citation-card
- Check that my code aint whack 

THANK YOU 🙏
